app/server/tasks/sendMail.js

```diff
@@ -0,0 +1,78 @@
+const nodemailer = require('nodemailer');
+const createWelcomeMail = require('../../email_templates/welcome.js');
+const createConfirmationMail = require('../../email_templates/confirmation.js');
+const createAmendmentMail = require('../../email_templates/amendment.js');
+const dotenv = require('dotenv');
+dotenv.config();
+
+module.exports = async ({
+  type,
+  email,
+  firstName,
+  lastName,
+  facilityName,
+  operatorName,
+  status
+}) => {
+  const transporter = nodemailer.createTransport(
+    process.env.SMTP_CONNECTION_STRING
```

Comment:
throw an error if `SMTP_CONNECTION_STRING` is not defined in `.env` ?

Reply:
In the scope of another PR, can we explore https://github.com/mozilla/node-convict please? That will give us config errors and environment-specific defaults for free. Happy to explore other libs as well if anyone has a fave.